cmd/worker/main.go

```diff
@@ -173,6 +175,29 @@ func main() {
 		clusterAuthService, err := intClusterAuth.NewDexClusterAuthService(clusterSecretStore)
 		emperror.Panic(errors.Wrap(err, "failed to create DexClusterAuthService"))
 
+		scmProvider := v.GetString("cicd.scm")
```

Comment:
The worker does not use the global configuration values, so you have to define defaults in the `configure` method.

Reply:
Done.